apps/aehttp/rebar.config

```diff
@@ -0,0 +1,2 @@
+{erl_opts, [debug_info]}.
+{deps, []}.
```

Comment:
aehttp shouldn't have a separate rebar.config. I think you should add lines to get this application build to the main rebar.config.